### ISSUE TITLE
Please Release Version: 5.3.0

### ISSUE THREAD

**Issue by kanwei** [webjars/dropzone#10]

**Comment by jamesward** [comment]
I can't figure out where to get a release artifact for `5.3.0`.  Can you help me figure that out?

**Comment by kanwei** [comment]
Yeah, it's a bit strange how they do it, but the final artifact is on this page: 

https://gitlab.com/meno/dropzone/-/jobs/49184863/artifacts/file/dist/dropzone.js

**Comment by jamesward** [comment]
I saw that one but it isn't the same git commit as the `5.3.0` tag:
https://gitlab.com/meno/dropzone/pipelines

So I'd rather not release the WebJar pointed to that one as `5.3.0` - but could release it with the git commit as the tag.  LMKWYT
